Clarifying parts of “Creating a default community health file” (#9082)
github/docs: content/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file.md

## About default community health files

> You can add default community health files to a public repository called `.github`, that is owned by an organization or user account. You can put the files in the root of the repository or in the `docs` or `.github` folders.

Reviewer: On second thought... all repositories are owned by organizations or user accounts, so this is no longer necessary.

```suggestion
You can add default community health files to a public repository called `.github`, in the root of the repository or in the `docs` or `.github` folders.
```

> You can add default community health files by putting them inside a public repository called `.github`, that is owned by an organization{% ifversion fpt or ghae or ghes %} or user account{% endif %}. You can put these files in the root of the repository, and in subdirectories of this repository named `docs` and `.github`.

Reviewer: We can actually remove this versioning now since it's currently specifying all versions.

```suggestion
You can add default community health files to a public repository called `.github`, that is owned by an organization or user account. You can put the files in the root of the repository or in the `docs` or `.github` folders.
```
